src/main/java/mate/academy/spring/model/dto/request/ShoppingCartRequestDto.java

```diff
@@ -0,0 +1,35 @@
+package mate.academy.spring.model.dto.request;
+
+import java.util.List;
+import mate.academy.spring.model.Ticket;
+import mate.academy.spring.model.User;
+
+public class ShoppingCartRequestDto {
```

Comment:
You don't use this DTO in your controllers

Reply:
I using in ShoppingCartRequestMapper in method  fromDto(ShoppingCartRequestDto dto)